application/client.core/src/app/environment/components/sidebar/shell/running/component.ts

```diff
@@ -72,21 +73,13 @@ export class SidebarAppShellRunningComponent implements OnDestroy, OnInit {
         };
     }
 
-    public _ng_onContexMenu(event: MouseEvent, process: IPCMessages.IShellProcess) {
-        const items: IMenuItem[] = [
-            {
-                caption: 'Terminate',
-                handler: () => {
-                    this.service.terminate({ session: this._sessionID, guid: process.guid }, process.command ).catch((error: string) => {
-                        this._logger.error(error);
-                    });
-                },
-            },
-        ];
-        ContextMenuService.show({
-            items: items,
-            x: event.pageX,
-            y: event.pageY,
+    public _ng_onMouseOver(guid: string) {
+        this._ng_hoverGuid = guid;
+    }
+
+    public _ng_onTerminate(process: IPCMessages.IShellProcess) {
+        this.service.terminate({ session: this._sessionID, guid: process.guid }, process.command ).catch((error: string) => {
+            this._logger.error(error);
```

Comment:
It's important operation and user feedback also is important here. Let's add user notification in case of error